src/import/importd.c

```diff
@@ -661,6 +664,9 @@ static int manager_new(Manager **ret) {
         if (r < 0)
                 return r;
 
+        m->setup_subvol = true;
+        m->setup_quota = true;
```

Comment:
Please move them slightly in the above and use the structured initializer:
```c
m = new(Manager, 1);
if (!m)
        return -ENOMEM;

*m = (Manager) {
        .setup_subvol = true,
        .setup_quota = true,
};
```

Reply:
Fixed.